### PR TITLE
Add AKS chart and CI

### PR DESCRIPTION
**Pull request by cmurphy**
https://github.com/rancher/rancher/issues/32293

### CODE REVIEW

File: charts/aks-operator/values.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+aksOperator:
+  image:
+    repository: rancher/aks-operator
+    tag: v1.0.1-rc1
```
Comment (thedadams) [inline charts/aks-operator/values.yaml:7]: I think it's better to set this to a nonsense value to help indicate that it is overwritten. Something like `999` or `v0.0.0`.

**Review by thedadams — COMMENTED** [review]
One minor change.

File: charts/aks-operator/values.yaml (hunk 1 above)
Comment (cmurphy) [reply charts/aks-operator/values.yaml]: done

**Review by thedadams — APPROVED** [review]

**Review by mjura — APPROVED** [review]
LGTM

**Review by cbron — APPROVED** [review]
lgtm, but may need to wait for build permissions to tag it.

**Pull request merged**